tests/ipex/test_modeling.py

```diff
@@ -213,17 +213,19 @@ class IPEXModelForCausalLMTest(unittest.TestCase):
         "blenderbot-small",
         "bloom",
         "codegen",
+        "falcon",
         "gpt2",
         "gpt_neo",
         "gpt_neox",
         "mistral",
         "llama",
         "llama2",
         # "phi",
+        "distilgpt2",
```

Comment:
could we have test for already exported models (to make sure we keep compatibility with these models)

Reply:
I see all test models are not exported. Do we need to specific tests for `distilgpt2`?

Reply:
yes that would be great to have for all as we add more and more support